python/res/enkf/config_keys.py

```diff
@@ -123,3 +123,8 @@ class ConfigKeys:
     ECL_FILE         = _ecl_file()
     OUTPUT_FORMAT    = _output_format()
     INIT_FILES       = _init_files()
+    LICENSE_PATH     = "LICENSE_PATH"
```

Comment:
If these are defined in the C-code, please fetch the value from there instead of defining it directly.